### PR TITLE
Minor style changes

### PR DESCRIPTION
**Pull request by stanparker**
Adds padding to video player and puts the record medium selector input in line with the rest.

### CODE REVIEW

File: src/client/mtna/record/record-style.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
+video {
```
Comment (DavidSouther) [inline src/client/mtna/record/record-style.scss]: Move this inside record-viewer, "just in case"

File: src/client/mtna/record/record-style.scss [hunk 2]
```diff
@@ -7,6 +12,10 @@ record-viewer {
     user-select: none;
   }
 
+  .medium {
```
Comment (DavidSouther) [inline src/client/mtna/record/record-style.scss]: Don't need a new class; use `md-select[ng-model="state.record.medium"]`

File: src/client/mtna/record/record-style.scss [hunk 3]
```diff
@@ -12,7 +7,7 @@ record-viewer {
     user-select: none;
   }
 
-  .medium {
+  md-select[ng-model="state.record.medium"] {
     padding-bottom:24px;
```
Comment (DavidSouther) [inline src/client/mtna/record/record-style.scss]: Space after :, here and below.

File: src/client/mtna/record/record-style.scss [hunk 4]
```diff
@@ -7,6 +7,10 @@ record-viewer {
     user-select: none;
   }
 
+  md-select[ng-model="state.record.medium"] {
+    padding-bottom:24px;
```
Comment (DavidSouther) [inline src/client/mtna/record/record-style.scss]: We need a linter for these.

**Comment by stanparker** [comment]
Thanks! Couple other things

A. It looks like there's another line to add the parameters to the stringify, because it only writes with linebreaks on update, not on the original write.

B. It doesn't rewrite the backup file when adding video, but it does when adding images

C. Renaming a tape produces a weird result:
1. Create a new tape called "Rita 6" and click done
2. Open the Record up and attempt to rename it to "Rita 6 - abc"
3. Notice that the folder changes to rita_6_-\_abc, but the .json file is still called rita_6.json, and the "label" field in rita_6.json still has "Rita 6" for a value.
4. Also notice the following toast error `500 Error saving Rita 6 - abc: Error: ENOENT: no such file or directory, rename &#39;/var/archives/data/rita_6/rita_6.json&#39; -&gt; &#39;/var/archives/data/rita_6_-_abc/rita_6_-_abc.json&#39;<br> &nbsp; &nbsp;at Error (native)
OK
`

**Pull request merged**